docs/swagger/wforce_api.7.yml

```diff
@@ -6,8 +6,8 @@ info:
   contact:
     name: "Dovecot Oy"
   license:
-    name: "GPL2"
-    url: http://github.com/PowerDNS/weakforced/master/LICENSE
+    name: "GPL3"
```

Comment:
Actually it's not a license change, that was already done - this is just bringing the "name" inline with what is actually contained in LICENSE